platform/features/layout/src/LayoutController.js

```diff
@@ -58,14 +58,14 @@ define(
                 return copy;
             }
 
-            // Compute panel positions based on the layout's object model
-            function lookupPanels(ids) {
+            /**
+             * Compute panel positions based on the layout's object model.
+             * Defined as member function to facilitate testing.
+             * @private
+             */
+            LayoutController.prototype.layoutPanels = function layoutPanels (ids) {
```

Comment:
This assignment is rather funny since it's in the constructor for LayoutController; this method gets reassigned with a new closure scope every time a new LayoutController gets instantiated.

I would do one of:

1. Move outside of the constructor call; may need to store some additional properties on `this` at constructor time to support this
2. Assign as `this.layoutPanels = ` such that this varies per instance (and can still access `$scope` et al via closure scope.)
3. Just leave it as an inner function (as `lookupPanels` had been)